src/env/src/config-updater.js

```diff
@@ -259,6 +262,39 @@ function updateStep21_v1_0_0i(targetConfig, sourceConfig, configKey) {
     saveGlobals(targetConfig, targetGlobals);
 }
 
+function updateStep21_MultiRoutes(targetConfig, sourceConfig, configKey) {
```

Comment:
It looks a little disturbing that the function name is "21", but it's actually step 22.